sources/packages/backend/libs/utilities/src/esdc-utils.ts

```diff
@@ -29,6 +29,15 @@ export function getGenderCode(gender: string): string {
   return gender === "female" ? "F" : "O";
 }
 
+/**
+ * Returns phone number removing the hyphens.
+ * @param phone phone number to be formatted
+ * @returns phone number removing the hyphens in between. (Eg: 123-456-7890 => 1234567890)
```

Comment:
Maybe I am missing something but the task was not about removing possible dashes, it was about scanning a string and keeping only a number.
For instance `+1 (788) 999-7777` would be converted to `17889997777`.
Similarly, `####4*****^%$#@$^9` would be converted to `49`.